COOKBOOK-FIXERS.md

```diff
@@ -309,11 +312,11 @@ class RemoveCommentsFixer extends AbstractFixer
         $tokens = Tokens::fromCode($content);
 
         $foundComments = $tokens->findGivenKind(T_COMMENT);
-        foreach($foundComments as $index => $token){
+        foreach ($foundComments as $index => $token) {
```

Comment:
Indeed, I missed the sentence in the following section mentionning these CS issues. I will revert these changes